OpenSim/Simulation/RegisterTypes_osimSimulation.cpp

```diff
@@ -273,7 +272,8 @@ OSIMSIMULATION_API void RegisterTypes_osimSimulation()
     Object::renameType("MusclePoint",       "PathPoint");
     Object::renameType("MuscleViaPoint",    "ConditionalPathPoint");
     Object::renameType("MovingMusclePoint", "MovingPathPoint");
-    Object::renameType("MusclePointSet",    "PathPointSet");
+    Object::renameType("MusclePointSet", "PathPointSet");
+    Object::renameType("Appearance", "SurfaceAppearance");
```

Comment:
True @chrisdembia but the code to convert old Geometry in Body.cpp uses "Appearance" rather then SurfaceAppearance, it seems cleaner to treat these as two separate XML changes but I don't feel strongly about it. If we remove the rename then we have to go back and fix the conversion code accordingly.